src/main/resources/api/service_openapi.yaml

```diff
@@ -46,6 +46,27 @@ paths:
         500:
           description: Creation request error
           $ref: '#/components/responses/ErrorResponse'
+  '/api/v1/get':
+    post:
```

Comment:
This is kinda funky. The endpoint accepts a body so it can't get a `GET`.... would be open to re-working this but given the current requirements it's what I've chosen.

Reply:
Can this be something like /api/v1/get/{id}? I think the auth tokens usually get passed via header, Create was just kind of a special case because it needs both user auth token + folder manager's auth token.